protocol/src/main/protobuf/User.proto

```diff
@@ -98,3 +106,141 @@ message BitToUserHandshake {
   optional string errorId = 4;
   optional string errorMessage = 5;
 }
+
+/*
+ * Enum indicating the request status.
+ */
+enum RequestStatus {
```

Comment:
Although we haven't done this previously, I think we should use "UNKNOWN = 0" for enums to avoid problems [mentioned here](http://androiddevblog.com/protocol-buffers-pitfall-adding-enum-values/).